README.md

```diff
@@ -1,38 +1,19 @@
-# STARTER-team01
+# w24-4pm-2-team01
 
 Lab instructions: <https://ucsb-cs156.github.io/w24/lab/team01.html>
 
-* TODO: Update the following table, then delete this line that says "TODO".
+| Name         | GitHub Id                                           |  Service                    | Controller                |
+|--------------|-----------------------------------------------------|-----------------------------|---------------------------|
+| Bharat Kathi | [bk1031](https://github.com/bk1031)                 | `JokeQueryService`      | `JokeController`      |
```

Comment:
Please align this line (if that's what the instructions say)

Reply:
Reformatted the table, markdown renders correctly either way 👍